PopupBridge/src/test/java/com/braintreepayments/popupbridge/PopupBridgeTest.java

```diff
@@ -43,7 +45,7 @@
 @RunWith(RobolectricTestRunner.class)
 public class PopupBridgeTest {
 
-    private AppCompatActivity mActivity;
+    private FragmentActivity mActivity;
```

Comment:
Do we want to keep a test around that uses an `AppCompatActivity` in addition to the newly supported `FragmentActivity`?